packages/@react-stately/table/src/TableBody.ts

```diff
@@ -31,17 +31,36 @@ TableBody.getCollectionNode = function* getCollectionNode<T>(props: TableBodyPro
         }
 
         for (let item of items) {
-          yield {
-            type: 'item',
-            value: item,
-            renderer: children
-          };
+          let element = children(item);
```

Comment:
isn't this going to be expensive?

Reply:
potentially, I'll need to test with a section with many rows. Open to any suggestions since I don't think I can distinguish between the Rows and Sections without doing something like that

Reply:
I don't see much of a difference in performance in the table with many sections story, but this will be changing when we move over to RAC collections so IMO this is fine for now

Reply:
todo: add story with sections w/ 100+rows in each